modules/preview-react/pill/lib/usePillModel.tsx

```diff
@@ -0,0 +1,71 @@
+import {createEventMap, Model, ToModelConfig, useEventMap} from '@workday/canvas-kit-react/common';
+
+type PillState = {
+  maxWidth?: number | string;
+  disabled?: boolean;
+  variations?: 'read-only' | 'interactive' | 'removable';
```

Comment:
Thoughts on this being in state? It will be pretty rare for a consumer to tap into the state and they can't really do much with it. This is used to determine which pill to render in `Pill.tsx`

@NicholasBoll